hack/deploy-kubevirt-and-cdi.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e
+
+NAMESPACE=${1:-kubevirt}
+
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+EOF
+
+# Deploying kuebvirt
+LATEST_KUBEVIRT=$(curl -L https://api.github.com/repos/kubevirt/kubevirt/releases | \
+            jq '.[] | select(.prerelease==false) | .name' | sort -V | tail -n1 | tr -d '"')
+
+oc apply -n $NAMESPACE -f "https://github.com/kubevirt/kubevirt/releases/download/${LATEST_KUBEVIRT}/kubevirt-operator.yaml"
+oc apply -n $NAMESPACE -f "https://github.com/kubevirt/kubevirt/releases/download/${LATEST_KUBEVIRT}/kubevirt-cr.yaml"
+
+oc apply -n $NAMESPACE -f - <<EOF
```

Comment:
Nit: I believe the ConfigMap is the "legacy" way of configuring kubevirt. Feature gates are now configured in the CR, for example:
```
 spec:
  configuration:
    developerConfiguration:
      featureGates:
      - DataVolumes
      - CPUManager
      - LiveMigration
```

Reply:
@akrejcir can you confirm this?
otherwise, /lgtm

Reply:
It is like Jed saying https://pkg.go.dev/kubevirt.io/client-go/api/v1#DeveloperConfiguration

Reply:
Changed to use `featureGates`.